gradle/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF

```diff
@@ -14,10 +14,14 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.jface,
  biz.aQute.bndlib;bundle-version="3.0.0",
  org.eclipse.buildship.core;bundle-version="1.0.0",
- com.gradleware.tooling.model;bundle-version="0.5.0"
+ com.gradleware.tooling.model;bundle-version="0.5.0",
+ com.liferay.ide.project.core,
+ org.eclipse.sapphire.modeling
```

Comment:
Gradle core should not need sapphire. we need to redesign the object communication so this isn't necessary.